web/src/app/app.component.html

```diff
@@ -83,7 +83,7 @@
       </div>
       <div fxHide.gt-xs fxFlex="100" fxLayout="row" fxLayoutAlign="space-between center" class="app-toolbar">
         <div>
-          <img src="assets/icons/Logo.svg" class="app-toolbar__logo"/>
+          <img src="assets/icons/logo.svg" class="app-toolbar__logo" />
```

Comment:
I think the `svg` file will need to be renamed too?

Reply:
I did but I forgot to push.